src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -643,9 +658,10 @@ private void DrawPackages(IEnumerable<INugetPackage> packages, bool canBeSelecte
 
             foreach (var package in packages)
             {
-                EditorGUILayout.BeginVertical(backgroundStyle);
-                DrawPackage(package, backgroundStyle, contrastStyle, canBeSelected);
-                EditorGUILayout.EndVertical();
+                using (new EditorGUILayout.VerticalScope(backgroundStyle))
```

Comment:
Is it really supported in Unity 2018? Because the doc doesn't list it:
![image](https://github.com/GlitchEnzo/NuGetForUnity/assets/53140583/25986f55-1224-4f21-a933-866d1b97f876)
But the CI should have failed if it is not supported.

Reply:
It does work, I am testing all of the code in Unity 2018.4.36f1 :)